src/FeatureFlags/FeatureFlagsServiceProvider.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Give\FeatureFlags;
+
+use Give\FeatureFlags\OptionBasedFormEditor\OptionBasedFormEditor;
+use Give\FeatureFlags\OptionBasedFormEditor\Settings\Advanced;
+use Give\FeatureFlags\OptionBasedFormEditor\Settings\DefaultOptions;
+use Give\FeatureFlags\OptionBasedFormEditor\Settings\General;
```

Comment:
Below, when we reference `General::class`, it isn't clear that this is referring to the General Settings.

We might want to use an alias to better describe this class.

```suggestion
use Give\FeatureFlags\OptionBasedFormEditor\Settings\General as GeneralSettings;
```

Reply:
Resolved in 224c913b94d5cefeced8c81ee4b1a6c33da5a222